documentation.md

```diff
@@ -54,6 +54,10 @@ Totally hides the widget if set to true and no plan matches the purchase amount.
 
 Allow to choose which payment plan's tab will be displayed by default. It will have effect only if the selected plan is eligible. If an array is provided, it will select the first eligible plan from this array.
 
+- hideBorder: `boolean` [optional, default: true]
```

Comment:
I saw a default value of `false` in the react component. I think the code is correct but the doc is wrong.